tgui/packages/tgui/interfaces/Hacking.js

```diff
@@ -0,0 +1,70 @@
+import { toArray } from 'common/collections';
+import { useBackend } from '../backend';
+import { AnimatedNumber, Section, Box } from '../components';
+import { formatMoney } from '../format';
+import { resolveAsset } from '../assets';
+import { Window } from '../layouts';
+
+export const Hacking = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    timeleft,
+    games = [[[]]],
+  } = data;
+  return (
+    <Window
+      width={1000}
+      height={1000}
```

Comment:
This UI sizes are illegal. Go straight to jail.

Reply:
each tile is 100px by 100px , meaning i can at best reduce the width down to 800, since the board is 8 x 8.

Reply:
Maximum allowed height is 768px (common resolution in cheap laptops) - 40px (windows taskbar) = 728px.